docs: Triggering Updates Using SSE and Mercure

dunglas opened the pull request:
Add an entry explaining how to plug a stream source using SSE and [Mercure](https://mercure.rocks).

dhh commented:
It's great that an integration like this is possible. And I'd be happy to link to it in the docs, but it's gotta be something much more muted. We can list it as an option with a two-line summary in one place, then link to a place that has more info about it ✌️

guillaumebriday commented on docs/handbook/04_streams.md:
```diff
@@ -149,3 +149,71 @@ Of all the techniques that are included with Turbo, it's with Turbo Streams you'
+
+export default class extends Controller {
+  connect() {
+    this.es = new EventSource(this.element.getAttribute("data-url"));
```
Maybe you could use the new values API as example: https://stimulus.hotwire.dev/reference/values?

dunglas commented:
@dhh I moved the tutorial to [Mercure's community pages](https://mercure.rocks/docs/ecosystem/hotwire) and  I added a summary in the *Integration with Server-Side Frameworks*  section.
